eng/pipelines/swagger-apiview.yml

```diff
@@ -0,0 +1,35 @@
+parameters:
+  - name: AzureSdkForNetDevOpsFeed
+    type: string
+    default: 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json'
+  - name: SwaggerApiParserVersion
+    type: string
+    default: '1.3.0-dev.20241029.2'
+
+trigger: none
+
+jobs:
+- job:
+  pool:
+    name: azsdk-pool-mms-ubuntu-2204-general
+    vmImage: ubuntu-22.04
+
+  steps:
+  - pwsh: |
+      dotnet tool install Azure.Sdk.Tools.SwaggerApiParser `
+        --version ${{ parameters.SwaggerApiParserVersion }}  `
+        --add-source ${{ parameters.AzureSdkForNetDevOpsFeed }} `
+        --global
+    displayName: Install APIView Swagger Parser
+
+  - pwsh: |
+      $(Build.SourcesDirectory)/eng/scripts/Swagger-APIView.ps1 `
+        -TempDirectory "$(Agent.TempDirectory)" `
+        -ArtiFactsDirectory "$(Build.ArtifactStagingDirectory)"
+    displayName: Generate Swagger APIView Tokens
+    ignoreLASTEXITCODE: true
```

Comment:
Should we set to false to respect the script exit code as the outcome of this step, to ensure that any issues are properly reflected in the build status.